File: string/ReverseString_1.java
public class ReverseString {
    public void reverseString(char[] s) {
        if(s.length <= 0)
            return;

        char temp;
        for (int i = 0; i < s.length; i ++ ) {
            temp = s[i];
            s[i] = s[s.length - 1 -i];
            s[s.length - 1 - i] = temp;

            if (s.length / 2 <= i + 1) {
                break;
            }
        }
    }

}

File: string/ReverseString_1.py
class Solution:
    def reverseString(self, s: List[str]) -> None:
        """
        Do not return anything, modify s in-place instead.
        """
        if (len(s) == 0):
            return

        i = 0
        while i < len(s):
            temp = s[i]
            s[i] = s[len(s) - 1 - i]
            s[len(s) - 1 - i] = temp

            if (len(s) / 2 <= i + 1):
                break
            else:
                i = i + 1
        